commcare_connect/opportunity/tables.py

```diff
@@ -116,6 +116,13 @@ def render_footer(self, bound_column, table):
         return sum(1 if bound_column.accessor.resolve(row) else 0 for row in table.data)
 
 
+class SumColumn(columns.Column):
+    def render_footer(self, bound_column, table):
+        return sum(
+            getattr(x, bound_column.accessor) for x in table.data if getattr(x, bound_column.accessor) is not None
+        )
```

Comment:
Below might give you same result with more conciseness.
```suggestion
        return sum(
            getattr(x, bound_column.accessor, 0) for x in table.data
        )
```

Reply:
@sravfeyn changed